library/modules/bigip_wait.py

```diff
@@ -23,6 +23,18 @@
     of rebooting due to a configuration change.
 version_added: 2.5
 options:
+  type:
+    description:
+      - The type of the BIGIP.
+      - Defaults to standard, the other choice being VCMP.
+      - The choice made defines what module/service Ansible will look for to establish
+      - that the device has recovered, so please ensure the correct choice is
+      - specified, specially when running this against VCMP.
```

Comment:
No need for hyphens, if this is a single passage you just need to use full stops and place the sentences one after the other, example here: https://github.com/F5Networks/f5-ansible/blob/devel/library/modules/bigip_monitor_dns.py#L79

Also if you intend to keep `/` you need to wrap the entire string with double quotes otherwise YAML generators will go nuts.

Reply:
Removed the /
Thanks for the link to the example. I was wondering about that, but took the safe option and put hyphens everywhere!